matrix-sdk-android/src/main/java/im/vector/matrix/android/api/session/crypto/CryptoService.kt

```diff
@@ -98,7 +98,9 @@ interface CryptoService {
 
     fun removeRoomKeysRequestListener(listener: GossipingRequestListener)
 
-    fun getDevicesList(callback: MatrixCallback<DevicesListResponse>)
+    fun fetchDevicesList(callback: MatrixCallback<DevicesListResponse>)
+    fun getMyDeviceInfo() : List<DeviceInfo>
```

Comment:
Please rename to getMyDevicesInfo (add an `s`) because it return a list.

Reply:
done